src/lib/utils/constants.js

```diff
@@ -0,0 +1,5 @@
+export const API_URL = "https://conduit.productionready.io/api"
```

Comment:
To mozesz sobie trzymac w `.env`. Tylko pamietaj ze `.env` powinien byc w .gitignore